From 3b2dcc0d6dcda4770541927ab4d866d66dc99574 Mon Sep 17 00:00:00 2001
From: Amit Kumar Mahapatra <amit.kumar-mahapatra@xilinx.com>
Date: Wed, 16 Sep 2020 14:24:52 -0600
Subject: [PATCH 1596/1851] mtd: rawnand: pl353: Initialized variable before
 using it

commit aa8ef55dd2bb2c56c04807b7e714146f9b17a4a7 from
https://github.com/Xilinx/linux-xlnx.git

A variable was used uninitialized. Fixed it by Initializing
the variable before using it.

Addresses-Coverity: "uninit_use: Using uninitialized value"
Event var_decl:	Declaring variable "val" without initializer

Signed-off-by: Amit Kumar Mahapatra <amit.kumar-mahapatra@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/mtd/nand/raw/pl353_nand.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mtd/nand/raw/pl353_nand.c b/drivers/mtd/nand/raw/pl353_nand.c
index 936cb78e35ce..27dfb4961e32 100644
--- a/drivers/mtd/nand/raw/pl353_nand.c
+++ b/drivers/mtd/nand/raw/pl353_nand.c
@@ -1125,7 +1125,7 @@ static int pl353_nand_probe(struct platform_device *pdev)
 	struct resource *res;
 	struct device_node *np, *dn;
 	struct clk *mclk;
-	u32 ret, val;
+	u32 ret, val = 0;
 
 	xnfc = devm_kzalloc(&pdev->dev, sizeof(*xnfc), GFP_KERNEL);
 	if (!xnfc)
-- 
2.31.1

